### ISSUE TITLE
misc not provided 

### ISSUE THREAD

**Issue by corymickelson** [rust-lang/rust-mode#95]
Error 'misc' is not provided, error thrown when opening *.rs files. I am assuming this is referring to line 13 in rust-mode.el. Any help is welcome, Thanks

**Comment by MicahChalmer** [comment]
That's odd.  The `misc.el` file has been provided with emacs for a long time, and should be available by default.  Do other packages work OK in your installation?  What version of emacs is it?  What OS?

**Comment by corymickelson** [comment]
I am running mac 10.10.5, emacs 24.5.1, I have never seen this error message with any other packages, but I honestly don't know which other packages require 'misc' either.  


     On Friday, August 28, 2015 11:41 PM, Micah Chalmer <notifications@github.com> wrote:
   

 That's odd. The misc.el file has been provided with emacs for a long time, and should be available by default. Do other packages work OK in your installation? What version of emacs is it? What OS?—

**Comment by no-reply** [comment]
I'm also seeing this issue with emacs 24.5.1 on an Arch Linux build.

**Comment by no-reply** [comment]
I was able to resolve this with `(provide 'misc)`. In my case, I added it to `init.el`.

**Comment by MicahChalmer** [comment]
Ok, so now two people saw the same thing and it has made me really curious...but I still couldn't reproduce it.  I tried running a fresh VM with emacs on arch linux and rust-mode - worked fine.  I also got emacs 24.5.1 for OS X downloaded from http://emacsformacosx.com/ - worked fine.

On the other hand...it turns out that the `(require 'misc)` is not needed anyway.  It was added in commit 3301c70 to get the `forward-to-word` function.  But the use of `forward-to-word` was removed shortly thereafter in 38dab12b.  So if `misc` is causing trouble it can just be removed.  But I still think those who are seeing this have an odd configuration that I don't understand.
